base.yaml

```diff
@@ -123,22 +95,21 @@ catalog_server:
 
 postgresql:
   _install: true
+  replication:
+    enabled: false
   persistence:
     size: 8Gi
 
 management_portal:
   _install: true
   replicaCount: 1  # should be 1
   postgres:
-    password: password
+    host: postgresql
```

Comment:
Why not change this value from the chart?
https://github.com/RADAR-base/RADAR-Kubernetes/blob/master/charts/management-portal/values.yaml#L64